src/Pages/ControlPanel.jsx

```diff
@@ -25,24 +25,46 @@ const ControlPanel = () => {
 	const isSmall = useMediaQuery(theme.breakpoints.down("md"));
 
 	const [ctrlVals, dispatchCtrlVals] = useReducer(ctrlValsReducer, { remote: null, local: {} });
-	const fetchRemoteValues = (discardLocal) => {
-		dispatchCtrlVals({ type: "discard_remote" });
-		Promise.all(
-			systemControlsCollections.map(collName => getDocs(collection(db, collName)))
-		).then(querySnapshotArray => {
-			// get an array of arrays of [path, data] pairs, then flatten it to a array of [path, data] pairs
-			const pairs = querySnapshotArray.map(querySnapshot =>
-				// get an array of [path, data] pairs
-				querySnapshot.docs.map(doc => [doc.ref.path, doc.data()])
-			).flat();
-			if (discardLocal) {
-				// do this first to avoid the unnecessary checks in replace_remote
-				dispatchCtrlVals({ type: "discard_local" });
-			}
-			dispatchCtrlVals({ type: "replace_remote", newRemote: Object.fromEntries(pairs) });
-		});
-	};
-	useEffect(() => fetchRemoteValues(), []);
+
+  const fetchRemoteValues = (discardLocal) => {
+    useEffect(() => {
+      dispatchCtrlVals({ type: "discard_remote" });
+
+      
+      //this is the listener for the realtime updates
+      systemControlsCollections.map((collName) => {
+        const collectionRef = query(collection(db, collName));
+        console.log(collName)
+          const unsubscribe = onSnapshot(collectionRef, (querySnapshot) => {
+            const pairs = querySnapshot.docs.map(doc => [doc.ref.path, doc.data()]);
+            if (discardLocal) {
+              // do this first to avoid the unnecessary checks in replace_remote
+              dispatchCtrlVals({ type: "discard_local" });
+            }
+            dispatchCtrlVals({ type: "replace_remote", newRemote: Object.fromEntries(pairs) });
```

Comment:
Ok so you are half right.

I thought the same thing initially––that all other values would grey out. but then it DID work (or so i thought) when I tested it. ie you could change individual values and the rest would remain. great! butttt actually upon further investigation you are still right :( it's just more subtle and so slipped past me: 

basically if i wanted to change a value in tolerances: upon receiving an updated control panel value, the other documents in tolerances stay the same (so it SEEMS like it works correclty). 

but the rest of the collections covered by control panel (lights and basking) are blanked out. This is because the onSnapshot queries each collection separately, and you can't query multiple collections at the same time.

I didn't notice the other sections being greyed out. so back to the drawing board for this rip